### PR TITLE
DOC: Add whatsnew for isnan and friends.

### PR DESCRIPTION
**Pull request by ssanderson**

### CODE REVIEW

File: docs/source/whatsnew/0.8.4.txt [hunk 1]
```diff
@@ -19,6 +19,8 @@ Enhancements
   passed to :class:`~zipline.algorithm.TradingAlgorithm` by the keyword argument
   ``create_event_context`` (:issue:`828`).
 
+* Added `isnan`, `notnan`, and `isfinite` methods to `Factor` (:issue:`861`).
```
Comment (llllllllll) [inline docs/source/whatsnew/0.8.4.txt]: rst uses two backquotes for monospace. Also, this should be a class reference

Comment (llllllllll) [inline docs/source/whatsnew/0.8.4.txt]: side note, rst-mode mode is really pretty

Comment (ssanderson) [inline docs/source/whatsnew/0.8.4.txt]: Is there a reason the whatsnew files are .txt and not .rst?

Comment (ssanderson) [inline docs/source/whatsnew/0.8.4.txt]: (It's a big annoying to have to M-x rst-mode.)

Comment (llllllllll) [inline docs/source/whatsnew/0.8.4.txt]: This is because they should not be pulled in by sphinx, they are just `.. include::`ed in the releases.rst page.

**Comment by ssanderson** [comment]
Closed in favor of https://github.com/quantopian/zipline/pull/864.

**Pull request closed without merging**